Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.5.5-jessie
```

Comment:
Like for `annotations`, we start from a Debian image; specialized for Python 3.5 in this case